### DISCUSSION TITLE
i have problem with messages on conversationID. My agent does not have memory

### DISCUSSION THREAD

**Discussion by diogo-salves** [langgenius/dify#8974] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

I'm using an application with n8n and dify for an agent. Everything was working normally, until I updated to version 0.9.1
now every message that arrives to my agent (with the same conversationID) is as if it were the first, my agent and I cannot see the message history, they do not exist, so for the agent, every message that arrives is as if it were the first and he has no memory.

![Captura de tela 2024-10-01 111748](https://github.com/user-attachments/assets/65f95c4b-2803-4c51-8269-14b70204a9c9)

**Comment by ip2cloud** [comment]
I'm having this same problem. Did you have a solution?

**Comment by nicklvs1307** [comment]
Tambem estou com o mesmo problema, alguem conseguiu resolver?

**Reply by ip2cloud** [reply]
ja ajustaram na 0.9.2 e na 0.10.2
